plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -76,8 +76,8 @@ test.describe( 'Add New Variable Product Page', () => {
 	} );
 
 	test( 'shows the variable product tour', async ( { page } ) => {
-		await page.goto( productPageURL );
-		await page.selectOption( '#product-type', 'variable', { force: true } );
```

Comment:
We really want to avoid the use of `force: true` as much as possible.  It bypasses all of Playwright's checks for readiness/stability.